htdocs/adherents/list.php

```diff
@@ -87,7 +87,6 @@
 
 // List of fields to search into when doing a "search in all"
 $fieldstosearchall = array(
-    'd.rowid'=>'Ref',
```

Comment:
It means we won't search anymore on the member Ref. This is annoying. What's wrong with this ? Is a sql error on postgresql ?